### DISCUSSION TITLE
Login idle timeout.

### DISCUSSION THREAD

**Discussion by ghost** [OrchardCMS/OrchardCore#14663] [Q&A] [answered]
I have a requirement of a 20 minute timeout for idle users.  I've updated program.cs with options.ExpireTimeSpan.  Currently I'm recreating the Admin theme to add js that will refresh the browser after 20 minutes.  Is this the standard (best practice)  way, or is there a better approach?

**Comment by hishamco** [comment, marked as answer]
I think it's good practice to show the login screen after a certain timeout. Adding a configuration for such a thing will be a plus.
